packages/next/src/server/config.ts

```diff
@@ -714,7 +714,7 @@ function assignDefaults(
       transform: 'react-bootstrap/{{member}}',
     },
     antd: {
-      transform: 'antd/es/{{member}}',
+      transform: 'antd/es/{{kebabCase member}}',
```

Comment:
Should it be `antd/lib/{{kebabCase member}}`?
We keep getting error on build with latest canary version

Reply:
Still the vercel build failing for antd imports with same error as below:
**Module not found: Can't resolve 'antd/es/Avatar'** 

Kindly fix this as we are waiting for our production deployment build Since last 24 hrs, or if any workaround until then you can suggest?
Thanks in advance.